Week7/CurrentNews/app/src/main/java/com/florencenjeri/cocktailsrecipe/database/NewsRepository.kt

```diff
@@ -0,0 +1,19 @@
+package com.florencenjeri.cocktailsrecipe.database
+
+import androidx.lifecycle.LiveData
+import com.florencenjeri.cocktailsrecipe.App
+import com.florencenjeri.cocktailsrecipe.model.New
+import com.florencenjeri.cocktailsrecipe.model.Success
+
+class NewsRepository() {
+    fun fetchNews(): LiveData<List<New>> = App.newsDao.fetchNews()
+
+    suspend fun insertNews() {
```

Comment:
This is a bit confusing as you're calling `insertNews()` with no parameters. Maybe name this `loadNews()` or something similar!